src/io/ipc/read/stream.rs

```diff
@@ -191,7 +208,15 @@ pub fn read_next<R: Read>(
     }
 }
 
-/// Arrow Stream reader
+/// Arrow Stream reader.
+///
+/// Once a stream is created the recommended way to interact with it is by
+/// iterating over its data. Each such iteration yields an [`Option<StreamState>`](StreamState),
+/// signaling that the stream might not currently have any data, even though it's still a "live"
+/// stream.
+///
+/// For a full usage examples consult [this](https://github.com/jorgecarleitao/arrow2/tree/main/examples/ipc_pyarrow)
+/// example in the main repository.
```

Comment:
```suggestion
/// For a full usage examples consult [this example](https://github.com/jorgecarleitao/arrow2/tree/main/examples/ipc_pyarrow).
```

(no need discuss git repositories here ^_^)